README.md

```diff
@@ -11,23 +11,32 @@ Interface to FreeType to generate font bitmaps run time
 - Add library: `freetype` (for GCC: `-L/usr/local/lib -lfreetype`)
 
 ## Usage in LVGL
+
+To enable cache, set`LV_USE_FT_CACHE_MANAGER  1`in lv_freetype.h.
+
 ```c
-lv_freetype_init(64); /*Cache max 64 glyphs*/
+/* init freetype library */
+lv_freetype_init(64, 1, 0);
```

Comment:
Could you add a little bit more comment here about what 64, 1, 0 means?

Reply:
Interface specification added to lv_freetype.h